HomeAssistant.py

```diff
@@ -1143,7 +1222,8 @@ def processYesOrNoResponse(self, session):
 					sessionId=session.sessionId,
 					text=self.randomTalk(text='userSaidNo', replace=[text]),
 					intentFilter=['UserRandomAnswer'],
-					currentDialogState=currentState
+					currentDialogState=currentState,
+					probabilityThreshold=0.1
```

Comment:
shouldn't not be required for RandomAnswer

Reply:
the probabilityThreshold ? i feel it is and in one occassion had to drop it to 0,0 for it to work *shrug*